### PR TITLE
Feature/health endpoint

### PR DESCRIPTION
**Pull request by alfre2v**
Please take a look but don't merge it to master too fast, as this changes a lot of important logic.

### CODE REVIEW

File: zmon_worker_monitor/web_server/web.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+def get_logger():
+    return logging.getLogger(__name__)
+
+
+@app.route('/')
+def root():
+    return 'Hello World!'
+
+
+@app.route('/api/running_procs/')
```
Comment (hjacobs) [inline zmon_worker_monitor/web_server/web.py]: can we remove the "/api/" prefix?

File: setup.py [hunk 2]
```diff
@@ -67,7 +67,7 @@ def load_req(fn):
 
             entry_points={
                 'console_scripts': [
-                    'zmon-worker = zmon_worker_monitor.web:main',
+                    'zmon-worker = zmon_worker_monitor.main:main',
```
Comment (hjacobs) [inline setup.py]: :+1: for this change

File: zmon_worker_monitor/web_server/web.py (hunk 1 above)
Comment (alfre2v) [inline zmon_worker_monitor/web_server/web.py]: Yes, although I was thinking in having two subtrees /api/ and /html/
Under /html/ we could put some pages for humans... maybe just one index page with a quick view of the state of the system, performance graphics, and problems?

Comment (hjacobs) [inline zmon_worker_monitor/web_server/web.py]: I would rather have a clean "REST" API, the root ("/") could serve some minimal HTML as an entry point for humans. Our Zalando API Guidelines also discourage using a "/api/" prefix (ZMON Controller uses an api prefix, but ZMON predates our API efforts).

**Comment by hjacobs** [comment]
:+1:

**Comment by hjacobs** [comment]
:+1:

**Pull request merged**